Automattic/jetpack
# My Jetpack: Update `ProductDetailCard` to use components and theme variables

Refactor `ProductDetailCard` to use style global variables and components (`Container`, `Col` and `Text`), keeping the movement for `component-driven` approach.

⚠️ **There's some changes on `spacings` and `fonts`, most of them are to make `ProductDetailCard` aligned with our RNA guidelines:** PbJqbW-5g-p2

#### Changes proposed in this Pull Request:

* Update `ProductDetailCard` styles and main code to use more components and css variables.
* Remove `ProductDetail` component since it became useless.
* Update `ProductInterstitial` to use only `ProductDetailCard` and apply borders on it.

#### Jetpack product discussion


N/A

#### Does this pull request change what data or activity we track or use?


No.

#### Testing instructions:






* Start `My Jetpack`.
* Navigate to products interstitials (`Add Anti-Spam`, `Add-Backup`,  etc...)
* It should have minimal style changes (most of them applying spacings based on our 8px rule).

#### Demo

![Screen Shot 2022-03-23 at 13 17 08](https://user-images.githubusercontent.com/1663717/159750735-30561cff-30e7-4192-a0b2-690c0b031476.png)

![Screen Shot 2022-03-23 at 13 20 31](https://user-images.githubusercontent.com/1663717/159750745-1c094bf6-77a2-43e4-986d-aaa12f812a37.png)

![Screen Shot 2022-03-23 at 13 28 41](https://user-images.githubusercontent.com/1663717/159750749-9612e317-e19d-4433-976b-aae74561a592.png)

![Screen Shot 2022-03-23 at 13 39 42](https://user-images.githubusercontent.com/1663717/159750930-3dcb7082-28e6-46d7-8659-03252ddd943e.png)

![Screen Shot 2022-03-23 at 13 39 49](https://user-images.githubusercontent.com/1663717/159750938-8816dee9-60c1-4b64-bfe4-7c6e6d1d3b7e.png)

![Screen Shot 2022-03-23 at 13 40 07](https://user-images.githubusercontent.com/1663717/159750948-a8e4a175-532d-4f4f-b42e-f146f0028f8f.png)
